exams/1_test.py

```diff
@@ -0,0 +1,43 @@
+# 1. Preberi DNK iz "dna1.txt".
+# 2. Izpisi prvih in zadnjih 12 znakov DNK.
+# 3. Deseti in enajsi nukleotid v zaporedju nakazujeta mesto prereza. Mesto se imenuje XY.
+# 4. Vsako zaporedje GGG v DNK med dvema nukleotidoma XY zamenjaj z zvezdico. --> Iščemo XYGGGXY, samo GGG se zamenja z zvezdico (pri vseh CTGGGCT).
+# 5. Izpisi mesto prve in druge zvezdice v DNK.
+# 6. Izpisi stevilo zvezdic.
+# 7. Odstrani zvezdice iz zaporedja in poisci razliko v dolzini zacetnega in koncnega DNK.
+# 8. Koncno zaporedje shrani v datoteko "new_dna1.txt".
+
+dna = open("dna1.txt", "r")
+dna = dna.read().strip().upper()
```

Comment:
Tukaj bi lahko dodala `old_len = len(dna)` in ti ne bi bilo treba racunati v 37. vrstici

Reply:
Ok 🥇 Si bom zapomnila za naslednjič.